qcodes/instrument_drivers/QDev/QDac.py

```diff
@@ -0,0 +1,484 @@
+        \n
+        8\t  0.000000\t\tX 1\t\tpA\n
+        7\t  0.000000\t\tX 1\t\tpA\n
+        ... (all 48 channels like this in a somewhat peculiar order)
+        (no termination afterward besides the \n ending the last channel)
+        returns a list of dicts [{v, vrange, irange}]
+        NOTE - channels are 1-based, but the return is a list, so of course
+        0-based, ie chan1 is out[0]
+        '''
+
+        # Status call
+
+        version_line = self.ask('status')
+
+        if version_line.startswith('Software Version: '):
+            self.version = version_line.strip().split(': ')[1]
+        else:
+            self._wait_and_clear()
+            raise ValueError('unrecognized version line: ' + version_line)
+
+        header_line = self.read()
+        headers = header_line.lower().strip('\r\n').split('\t')
+        expected_headers = ['channel', 'out v', '', 'voltage range',
+                            'current range']
+        if headers != expected_headers:
+            raise ValueError('unrecognized header line: ' + header_line)
+
+        chans = [{} for i in self.chan_range]
+        chans_left = set(self.chan_range)
+        while chans_left:
+            line = self.read().strip()
+            if not line:
+                continue
+            chanstr, v, _, vrange, _, irange = line.split('\t')
+            chan = int(chanstr)
+            chanstr = '{:02}'.format(chan)
+
+            irange_trans = {'hi cur': 1, 'lo cur': 0}
+
+            vals_dict = {
+                'vrange': ('ch{}_vrange',
+                           self.voltage_range_status[vrange.strip()]),
+                'irange': ('ch{}_irange', irange_trans[irange]),
+                'v': ('ch{}_v', float(v)),
+            }
+
+            chans[chan - 1] = vals_dict
+            for param in vals_dict:
+                parameter = vals_dict[param][0].format(chanstr)
+                value = vals_dict[param][1]
+                if param == 'vrange':
+                    attenuation = 0.1*value
+                if param == 'v':
+                    value *= attenuation
+                self.parameters[parameter]._save_val(value)
+            chans_left.remove(chan)
+
+        if readcurrents:
+            for chan in range(1, self.num_chans+1):
+                paramname = 'ch{:02}_i'.format(chan)
+                param = self.parameters[paramname]
+                param._save_val(param.get())
+
+        self._status = chans
+        self._status_ts = datetime.now()
+        return chans
+
+    def _setslope(self, chan, slope):
+        """
+        set_cmd for the chXX_slope parameter, the maximum slope of a channel.
+
+        Args:
+            chan (int): The channel number (1-48)
+            slope (Union[float, str]): The slope in V/s. Write 'Inf' to allow
+              arbitraryly small rise times.
+        """
+        if chan not in range(1, 49):
+            raise ValueError('Channel number must be 1-48.')
+
+        if slope == 'Inf':
```

Comment:
Should perhaps also allow for the inf float

`if slope == 'Inf' or type(slope) == float and math.isinf(slope)`

We could also only use the float inf but that is a bit tricky since you will have to use numpy as in `numpy.inf`